crates/common/network_spec/src/networks.rs

```diff
@@ -72,6 +78,8 @@ pub static SEPOLIA: LazyLock<Arc<NetworkSpec>> = LazyLock::new(|| {
             genesis_fork_version: fixed_bytes!("0x90000069"),
         },
         deposit_contract_address: address!("0x7f02C3E3c98b133055B8B348B2Ac625669Ed295D"),
+        domain_aggregate_and_proof: "0x06000000".to_string(),
+        inactivity_penalty_quotient: INACTIVITY_PENALTY_QUOTIENT_BELLATRIX,
```

Comment:
I meant I think you should remove these 2 fields from the NetworkSpec struct, and just sub them in how you did it originally,

Except make a constant for `pub const DOMAIN_AGGREGATE_AND_PROOF: B32 = fixed_bytes!("0x06000000");`